xml/security_cryptofs.xml

```diff
@@ -269,4 +278,83 @@
     the file.
   </para>
  </sect1>
+ <sect1 xml:id="sec-security-cryptofs-rage">
+  <title>Encrypting files with Rage</title>
+  <para>
+   Rage is a secure file encryption software to encrypt files.
+   Rage has small explicit keys and does not have configuration options.
```

Comment:
This intro doesn't read very well? People will think small keys == insecure. Perhaps it should say "Rage has keys that are easy to exchange with other people, and has secure defaults to prevent accidental misuse or leaks of sensitive data".